### ISSUE TITLE
Tray Integration - Attachment Image

### ISSUE THREAD

**Issue by hudsonsecurity** [Fmstrat/pushover-desktop#1]
Would be awesome on Linux if you could add the attachment image to the system notification or worse case scenario change the icon of pushover to the attached image.

Thanks great software!

**Comment by ozgurerdogan** [comment]
Yes, if you miss the sound notification, there is not more notify.
